#source/scss/style.scss

```diff
@@ -0,0 +1,98 @@
+@import "nullstyle.scss";
+
+
+ body{
+ 	margin: 0 auto;
+ 	font-family: 'Montserrat', sans-serif;
+ }
+ @import "../fonts/font.css";
+
+ // ======================================= header
+.header {
+	padding: 24px 0 0 0; 
+}
+.header__container {
+	display: flex; 
+	justify-content: space-between;
+	align-items: center;
+}
+.header__logo {
+	padding: 0 0 0 0;
+}
+.logo {}
+.header__menu {
+	width: 490px;

```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/chendler_tesla/pullRequest?prid=5803897)